content/html/concepts/tags/terms/doctype/doctype.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: '<doctype>' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: '<!DOCTYPE>'
```

It is valid to name this `<!doctype>` since this tag is not case-sensitive. However, let's keep this in caps per convention and we can make a **Note:** at the bottom of this entry if this is desired.